packages/vdom-extension/src/index.tsx

```diff
@@ -43,27 +45,71 @@ export class RenderedVDOM extends Widget implements IRenderMime.IRenderer {
     this.addClass('jp-RenderedHTML');
     this.addClass('jp-RenderedHTMLCommon');
     this._mimeType = options.mimeType;
+    // Get current kernel session (hack for mimerender extension)
+    this._session = (options.resolver as any)._session;
```

Comment:
I think this will fail if we open a `.vdom` file, which doesn't have any associated session. Can we check for a `null`/`undefined` session in that case?

Also, can you add a big honking TODO here that we should figure out a safer way to get the session in a rendermime extension? :)

Reply:
I'll do one better and create an issue: https://github.com/jupyterlab/jupyterlab/issues/5954

Reply:
We've typically been very shy about adding more API surface area to `rendermime-interfaces` since it makes the API less stable. As a result, it has stayed at 1.0 while services is at 4.0! If it ends up causing more headaches than it's worth, though, we may want to revisit it.

Reply:
Ya I understand that